local-setup/scripts/enable-notifications.sh

```
@@ -0,0 +1,959 @@
+  if [[ "$DRY_RUN" == true ]]; then
+    printf '   %s[dry-run]%s %s\n' "${C_YEL}" "${C_RESET}" "$cmd"
+    return 0
+  fi
+  eval "$cmd"
+}
+
+# require <desc> <predicate> — PRECONDITION. Abort the step/script if it fails.
+require() {
+  local desc="$1"; shift
+  if [[ "$DRY_RUN" == true ]]; then printf '   %s[dry-run]%s require: %s\n' "${C_YEL}" "${C_RESET}" "$desc"; return 0; fi
+  if eval "$@" >/dev/null 2>&1; then ok "precondition: ${desc}"; return 0
```

Shellcheck (0.11.0)

[warning] eval negates the benefit of arrays. Drop eval to preserve whitespace/symbols (or eval as string).

(SC2294)